Emit 'Changed' signal for CameraAttributesPhysical and CameraAttributesPractical properties that didn't

amarsero opened the pull request:
<!--
Please target the `master` branch in priority.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html
-->
Fixes #98784

ramokz commented:
Can confirm that the changes resolve the issue :tada:

akien-mga commented on scene/resources/camera_attributes.cpp:
```diff
@@ -217,6 +217,7 @@ void CameraAttributesPractical::_update_dof_blur() {
 			dof_blur_near_distance,
 			dof_blur_near_transition,
 			dof_blur_amount);
+	emit_changed();
```
This would be better called in each setter, even if it's a bit of code duplication.

`_update_dof_blur` is currently also called from the constructor, where it shouldn't emit changed unnecessarily, and it could be used in other places in the future which are not property setters - at least based on its name.

amarsero replied:
I thought about this, but then i saw [CameraAttributesPractical::_update_auto_exposure#L245](https://github.com/godotengine/godot/blob/6c5bb99bf5a6d2a5b94cf01e41c10727700ce76c/scene/resources/camera_attributes.cpp#L245) called `emit_changed()` there and did the same thing. Would you like me to fix `_update_auto_exposure` as well?

akien-mga replied:
Ah indeed. I'd say yes, let's fix `_update_auto_exposure` too. The other `_update_exposure` uses the pattern that I suggest, keeping the `emit_changed` only in the setter (and that one is also called in the constructor, unlike `_update_auto_exposure`).

amarsero committed:
Add emit_changed() for all  properties in CameraAttributes where it was missing, remove emit_changed() from private methods.

amarsero commented:
Added `emit_changed()` for all properties in CameraAttributes where it was missing and removed `emit_changed()` from private methods.
Also had to change the constructor for both since they used set properties to change themselves and we want to avoid calling `emit_changed()` on the constructor.
